README.md

```diff
@@ -25,39 +27,62 @@ const Fastify = require('fastify')
 const server = Fastify()
 
 server.register(require('fastify-http-proxy'), {
-  upstream,
-  prefix: '/upstream', // optional
+  'http://my-api.example.com',
+  prefix: '/api', // optional
   http2: false // optional
 })
 
 server.listen(3000)
 ```
 
-For a more complete example, see `example.js`.
+This will proxy any request starting with `/api` to `http://my-api.example.com`. For instance `http://localhost:3000/api/users` will be proxied to `http://my-api.example.com/users`.
+
+If you want to have different proxies on different prefixes in you can register multiple instances of the plugin as shown in the following snippet:
```

Comment:
If you register this plugin multiple times, probably the prefix handling should be required :)

Reply:
Good point, let me sure we capture this as it might be non-obvius!

Reply:
Let me know if you are ok with the clarification added with the latest commit